### PR TITLE
RS-105: Release 1.8.0

### PR DESCRIPTION
**Pull request by atimin**
Changes:

* Add a release note
* Release 1.8.x documentation

### CODE REVIEW

**Comment by atimin** [comment]
@AnthonyCvn will need your help to fix the build. Something with Azure

**Comment by AnthonyCvn** [comment]
> @AnthonyCvn will need your help to fix the build. Something with Azure

@atimin all good, it's fixed. The Azure static web app has limited staging environment (max 3), so I deleted one.

File: blog/2024-01-28-reductstore-v1_8_0-released/index.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+In this release, we've introduced a crucial feature for any database - data replication. Now, you can create a server-side task that "subscribes" to new records written to a bucket and forwards them to another bucket. This bucket can be located on the same instance or a remote one. Since all databases implement replication differently based on their specializations, let's examine how ReductStore tackles this.
+
+<!--more-->
+{/* truncate */}
+
+### Replication in ReductStore
+
+#### Append-Only Approach
+
+Currently, replication is append-only, meaning we only synchronize new records when they are written. As ReductStore is typically used as a large ring-buffer for blob data, it automatically removes old data when the disk quota is exceeded. Therefore, it doesn't make sense to replicate the remove operation on a remote instance where the disk space might be different.
```
Comment (AnthonyCvn) [inline blog/2024-01-28-reductstore-v1_8_0-released/index.md:27]: I would clarify the "we only synchronize new records when they are written" and already say here that new records are copied to the destination buckets but not deleted or updated.

Comment (AnthonyCvn) [reply blog/2024-01-28-reductstore-v1_8_0-released/index.md:27]: The "large ring-buffer" could also be clarified by mentionning that it's an efficient FIFO buffer

File: blog/2024-01-28-reductstore-v1_8_0-released/index.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+Our goal is to create a storage solution tailored for IoT and Edge computing, and this consideration influenced our approach to replication.
+
+ReductStore's replication uses a transaction log to persistently store record IDs for replication. If a remote instance is unavailable, the database retains these records in its logs. This method allows for later synchronization when the remote instance is restored.
+
+Since an IoT device may not always have a public IP for various reasons, ReductStore implements a "push" model. In this model, a source instance writes data to a destination instance, eliminating the need for direct access to the source device.
+
+#### Conditional Replication
+
+ReductStore does more than just mirror data; it also provides conditions based on labels to filter data before sending it to another instance. This type of data reduction helps reduce traffic and extend storage time for important data.
```
Comment (AnthonyCvn) [inline blog/2024-01-28-reductstore-v1_8_0-released/index.md:39]: I would add a small example to show how good this is

File: blog/2024-01-28-reductstore-v1_8_0-released/index.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+This is a perfect case for replication. We can create an additional bucket with a small quota and replicate only the anomalous images there. Depending on the quota chosen, it may store a few months' worth of data, which could be sufficient for our purposes.
+
+![Local Replication Example](./img/local-replication.webp)
+
+
+#### Remote Replication
+
+What if our device doesn't have a public IP, or the connection quality is too low for direct device interaction? We could address this by setting up a server with a ReductStore in our infrastructure and replicating the anomalous images from the edge device there.
+
```
Comment (AnthonyCvn) [inline blog/2024-01-28-reductstore-v1_8_0-released/index.md:57]: We could address this by setting up a server with a ReductStore database in our infrastructure and replicating the anomalous images from the edge device to the server.

File: blog/2024-01-28-reductstore-v1_8_0-released/index.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
+---
+title: "ReductStore v1.8.0 Has Been Released with Data Replication"
+description: ReductStore v1.8.0 introduces data replication between ReductStore instances.
+authors: alexey
+tags: [news]
+slug: news/reductstore-8-released
+date: 2024-01-28
+---
+
+# ReductStore v1.8.0 Has Been Released with Data Replication
```
Comment (AnthonyCvn) [inline blog/2024-01-28-reductstore-v1_8_0-released/index.md:10]: What do you think of "Reductstore V1.8.0 Released: Introducing Data Replication" ?

File: blog/2024-01-28-reductstore-v1_8_0-released/index.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+Since an IoT device may not always have a public IP for various reasons, ReductStore implements a "push" model. In this model, a source instance writes data to a destination instance, eliminating the need for direct access to the source device.
+
+#### Conditional Replication
+
+ReductStore does more than just mirror data; it also provides conditions based on labels to filter data before sending it to another instance. This type of data reduction helps reduce traffic and extend storage time for important data.
+
+### Typical Use Cases
+
+We will soon provide detailed information about use cases for replication. For now, let's briefly discuss two scenarios where replication can help simplify your application.
```
[the remark this answers is not in the document]
Comment (atimin) [reply blog/2024-01-28-reductstore-v1_8_0-released/index.md:43]: I've refrazed and remove the mention about future posts. I think it is irrelevant.

File: blog/2024-01-28-reductstore-v1_8_0-released/index.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+#### Local Replication
+
+In this use case, we have a computer vision application that detects product anomalies using images from a CV camera. The detection algorithm labels images as good or anomalous and stores them in the data bucket. We receive a few images per second, and the disk space is only sufficient to store a few days' worth of data. This amounts to a lot of images, and we would like to improve or validate our model with anomalous images. Unfortunately, anomalies occur very rarely, and we don't have enough images within our storage capacity.
```
[the remark this answers is not in the document]
Comment (atimin) [reply blog/2024-01-28-reductstore-v1_8_0-released/index.md:47]: You actually got it. Refrized, the new version should be better.

**Review by AnthonyCvn — APPROVED** [review]

**Pull request merged**